.github/workflows/goLint.yml

```diff
@@ -11,7 +11,7 @@ on:
       golangci-lint-version:
         required: false
         type: string
-        default: v1.64.4
+        default: latest
```

Comment:
This could bite us when new rules are activated, but let's see how it goes

Reply:
It's better, I think, to have the most recent version than one that hasn't been updated in quite a while.

We don't enable linters by default ([ref](https://github.com/smallstep/workflows/blob/1e9cc00b8d15eb9bc0ed53fa60235e2232d6301b/.golangci.yml#L158)).

The new changes also enforce configuration file verification, so that'd be a huge plus going forward!